entries/propEqual.xml

```diff
@@ -19,7 +19,7 @@
 	<longdesc>
 		<p>The <code>propEqual()</code> assertion provides strictly (<code>===</code>) comparison of Object properties. Unlike <code>deepEqual()</code>, this assertion can be used to compare two objects made with different constructors and prototype.</p>
 		<p><a href="/strictEqual"><code>strictEqual()</code></a> can be used to test strict equality.</p>
```

Comment:
@arthurvr  @arschmitz  The commit 4ccbf0e1dc solves all the remaining broke links together with the issues with versioning